Development

josephhalstead opened the pull request:
- add specialist codes for familial cancer team
- add new not analysed - unrelated to phenotype option
- auto load genes from OMIM

SophieS9 committed:
Small bug fix in CNV classification criteria

SophieS9 committed:
Adding reference to SOP and removing mention of SNV guidelines

josephhalstead committed:
add genes from omim and add not analysed extras

josephhalstead committed:
format

josephhalstead committed:
Merge pull request #56 from AWGL/cnv

CNV Database

SophieS9 reviewed (changes requested):
Ran unit tests and all worked with no errors. 

- Need to update acmg_questions.json file in the fixtures folder to be the most recent.
- Need to update readme to contain information about loading the gene_data.csv file with omim annotations. 
- Updating the inheritance pattern didn't appear to change the "Gene inheritance pattern" shown in the table above. 
- Is there any way of pre-populating the Conditions box in the edit gene info form with the information from omim? At the moment you can only add other conditions but can't edit the conditions from omim at all - is this deliberate? Would they need to do this?
- When I update the condition of a gene manually, this is then saved for that gene when it's re-uploaded again - just checking that we want this to be the case. 
- You've pushed the change in settings.py for the database to be local, needs to go back to cluster.

josephhalstead committed:
fixes for code review

josephhalstead committed:
update settings to be correct

josephhalstead commented:
Thanks for looking over this -

1) Fixtures updated
2) Readme updated
3) Is a feature not a bug - inheritance only updates if a new condition is added - cannot be added alone
4) Added view_gene_phenotypes, edit_gene_phenotypes and delete_gene_phenotypes so that users can edit/delete phenotypes. They have to click the hyperlink to do this
5) Yes - manually added phenotypes are retained even if OMIM updated
6) Fixed 

Any other issues let me know!

SophieS9 commented:
All comments/changes made and answered :) thank you! 

Re-ran unit tests with changes and still working fine.
